languages/rust/oso/tests/test_polar_rust.rs

```diff
@@ -454,13 +454,6 @@ fn test_unify_externals() {
 
     test.oso.register_class(bar_class).unwrap();
 
-    let mut query = test.oso.query("x = new Bar(1) = new Bar(2)").unwrap();
```

Comment:
Deleting these tests because they are now covered by parser tests: `x = y = z` parses to `(x = y)  = z` and you cannot unify a variable with an expression.